template/addons/trpc/auth-index-router.ts

```diff
@@ -3,12 +3,10 @@ import { createRouter } from "./context";
 import superjson from "superjson";
 
 import { exampleRouter } from "./example";
-import { authRouter } from "./auth";
 
 export const appRouter = createRouter()
   .transformer(superjson)
-  .merge("example.", exampleRouter)
-  .merge("auth.", authRouter);
```

Comment:
Wouldn't it be nice though to keep an example of a protected route using the protected router?

Reply:
+1 to this, looks great otherwise!